detox/src/devices/drivers/android/tools/DeviceHandle.js

```diff
@@ -0,0 +1,22 @@
+class DeviceHandle {
+  constructor(deviceString) {
+    const [adbName, status] = deviceString.split('\t');
+    this.type = this.inferDeviceType(adbName);
+    this.adbName = adbName;
+    this.status = status;
+  }
+
+  inferDeviceType(adbName) {
```

Comment:
Could you mark it by underscore convention as private (`_inferDeviceType`), please?

Reply:
Changed